.vscode/settings.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
@szymonjanota will this file override personal settings?

Reply:
Yes it is supposed to - so every dev gets same environment